packages/web3-errors/src/errors/contract_errors.ts

```diff
@@ -159,7 +159,7 @@ export class Eip838ExecutionError extends Web3ContractError {
 		//	 error.data, error.data.data or error.data.originalError.data (https://github.com/web3/web3.js/issues/4454#issuecomment-1485953455)
 		if (typeof error.data === 'object') {
 			let originalError: { data: string };
-			if ('originalError' in error.data) {
+			if (error.data && 'originalError' in error.data) {
```

Comment:
@Alfxjx Thanks, also add changelog with issue #, in :
https://github.com/web3/web3.js/blob/4.x/packages/web3-errors/CHANGELOG.md of your branch

Reply:
Thank you for your comments, I submitted the changelog, is it right?

Reply:
Thanks @Alfxjx ,
No need to update the version. We do the version update later in a separate PR.